SU2_CFD/src/solver_direct_elasticity.cpp

```diff
@@ -157,22 +157,26 @@ CFEASolver::CFEASolver(CGeometry *geometry, CConfig *config) : CSolver() {
   else if (nDim == 3) {
 
       element_container[FEA_TERM][EL_TETRA] = new CTETRA1(nDim, config);
-      element_container[FEA_TERM][EL_HEXA] = new CHEXA8(nDim, config);
+      element_container[FEA_TERM][EL_HEXA]  = new CHEXA8 (nDim, config);
+      element_container[FEA_TERM][EL_PYRAM] = new CPYRAM5(nDim, config);
+      element_container[FEA_TERM][EL_PRISM] = new CPRISM6(nDim, config);
 
       if (de_effects){
         element_container[DE_TERM][EL_TETRA] = new CTETRA1(nDim, config);
-        element_container[DE_TERM][EL_HEXA] = new CHEXA8(nDim, config);
+        element_container[DE_TERM][EL_HEXA]  = new CHEXA8 (nDim, config);
+        element_container[DE_TERM][EL_PYRAM] = new CPYRAM5(nDim, config);
+        element_container[DE_TERM][EL_PRISM] = new CPRISM6(nDim, config);
       }
 
       if (incompressible) {
         element_container[INC_TERM][EL_TETRA] = new CTETRA1(nDim, config);
-        element_container[INC_TERM][EL_HEXA] = new CHEXA1(nDim, config);
+        element_container[INC_TERM][EL_HEXA]  = new CHEXA1 (nDim, config);
```

Comment:
Incompressible will not work with Pyramids and Prisms for now but an error is thrown if someone tries it.

Reply:
Ok, I can do that, there are no implications to the other (than neo-Hooken) material models right?

Reply:
No, I think you would only have to remove the NeoHookean_Incomp. The Knowles model didn't use the under-integrated elements and is listed under NEARLY_INCOMPRESSIBLE_MAT, so it doesn't use the INC_TERM elements. That should simplify the code - and if someone wants to take it from there, we always have the older versions.